mmdeploy/codebase/mmcls/deploy/classification.py

```diff
@@ -224,13 +226,19 @@ def evaluate_outputs(model_cfg: mmcv.Config,
                 evaluation. It is useful when you want to format the result
                 to a specific format and submit it to the test server.
                 Default: False.
+            log_file (str | None): The file to write the evaluation results.
+                Defaults to `None` and the results will only print on stdout.
         """
         import warnings
+        from mmcv.utils import get_logger
+        logger = get_logger('test', log_file=log_file, log_level=logging.INFO)
+        print(log_file)
```

Comment:
Are these two line for debugging?

Reply:
Line 235 is for debugging! I will delete it.